daily-challenges/2024-10/15/khanh5923_15.spec.ts

```
@@ -0,0 +1,49 @@
+}
+
+function reasonablePrice(price) {
+  if (price < 25000000) {
+    console.log("Tôi đã đủ tiền mua");
+  }
+  else {
+    console.log("Tôi không đủ tiền mua");
+  }
+}
+test("daily/10/15_multiple-pages", async ({}) => {
+  const browser = await chromium.launch();
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)